### PR TITLE
feat(payment): PAYPAL-000 created storage package

### PR DESCRIPTION
**Pull request by serhii-tkachenko**
## What?
Created "storage" package and added BrowserStorage there.

## Why?
To be able to use it in different integration packages

## Testing / Proof
Unit tests
CI

### CODE REVIEW

**Review by PavlenkoM — APPROVED** [review]

**Comment by animesh1987** [comment]
Hey @serhii-tkachenko don't think we should move this to payment integration service. maybe a better idea is to move this a separate package?

**Comment by serhii-tkachenko** [comment]
> Hey @serhii-tkachenko don't think we should move this to payment integration service. maybe a better idea is to move this a separate package?

Hmm.. I think you are right. Let me update it

File: packages/storage/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# storage
```
Comment (animesh1987) [inline packages/storage/README.md:1]: Can you please update the readme to be a bit more detailed, compared to boilerplate one.

**Review by animesh1987 — APPROVED** [review]
Looks good, can you please address one comment before merging.

File: packages/storage/README.md (hunk 1 above)
Comment (serhii-tkachenko) [reply packages/storage/README.md:1]: Sure. Give me some time to update it

**Pull request merged**